packages/pie/src/Pie.tsx

```diff
@@ -31,8 +31,8 @@ const Pie = <RawDatum, >({
     innerRadius: innerRadiusRatio = defaultProps.innerRadius,
     cornerRadius = defaultProps.cornerRadius,
 
-    width,
-    height,
+    width = 0,
```

Comment:
Actually this is an error I made when migrating the `pie` package:

```typescript
export type CommonPieProps<RawDatum> = Dimensions & { /* ... */ }

export type PieSvgProps<RawDatum> = DataProps<RawDatum> &
    Partial<CommonPieProps<RawDatum>> &
    SvgDefsAndFill<ComputedDatum<RawDatum>> &
    PieHandlers<RawDatum, SVGPathElement> & {
        layers?: PieLayer<RawDatum>[]
    }
```

Dimensions should be required, removing them from common props and:

```typescript
export type PieSvgProps<RawDatum> = DataProps<RawDatum> &
    Partial<CommonPieProps<RawDatum>> &
    Dimensions &
    SvgDefsAndFill<ComputedDatum<RawDatum>> &
    PieHandlers<RawDatum, SVGPathElement> & {
        layers?: PieLayer<RawDatum>[]
    }
```